### DISCUSSION TITLE
NanoGPT Slowrun

### DISCUSSION THREAD

**Discussion by varunneal** [KellerJordan/modded-nanogpt#238] [General]
There's a new competition ([NanoGPT Slowrun](https://github.com/qlabs-eng/slowrun)) with the goal of minimizing validation loss within a small token budget (100M). The overall structure is pretty similar to Modded NanoGPT but it is essentially untimed (there is a 1 hour track and an unlimited track). I believe the initial record is forked from NanoChat and attains ~3.402 in 50 minutes. 

Curious what people in the community think. I think a lot of promising but unsuccessful ideas like MoE, expensive optimizers, full hyperconnections, and even non-gradient descent methods could be viable. It would also show what is theoretically attainable at just 100M tokens.
